src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,126 @@
+     *
+     * @param task The task added.
+     * @param size The number of tasks in the list.
+     */
+    public static void showAddTaskMessage(Task task, int size) {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println(INDENTATION + "Got it. I've added this task:");
+        System.out.println(INDENTATION + task.toString());
+        if (size > 1) {
+            System.out.println(INDENTATION + "Now you have " + size + " tasks in the list.");
+        } else {
+            System.out.println(INDENTATION + "Now you have " + size + " task in the list.");
+        }
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    /**
+     * Prints remove task message and the number of tasks in the list.
+     *
+     * @param task The task removed
+     * @param size The number of tasks in the list.
+     */
+    public static void showRemoveTaskMessage(Task task, int size) {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println(INDENTATION + "Got it. I've removed this task:");
+        System.out.println(INDENTATION + task.toString());
+        if (size > 1) {
+            System.out.println(INDENTATION + "Now you have " + size + " tasks in the list.");
+        } else {
+            System.out.println(INDENTATION + "Now you have " + size + " task in the list.");
+        }
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    /**
+     * Prints bye message.
+     */
+    public static void bye() {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println(INDENTATION + "Bye. Hope to see you again soon!");
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    /**
+     * Prints all the matching tasks in the given task list.
+     *
+     * @param taskList The given task list.
+     */
+    public static void showFindTasks(List<Task> taskList) {
+        System.out.println(Ui.HORIZONTAL_LINE);
+        System.out.println(Ui.INDENTATION + "Here are the matching tasks in your list:");
+        if (taskList.isEmpty()) {
+            System.out.println(Ui.INDENTATION + "No matching tasks");
+        } else {
+            for (int i = 0; i < taskList.size(); ++i) {
+                System.out.println(Ui.INDENTATION + (i + 1) + ". " + taskList.get(i).toString());
+            }
+        }
+        System.out.println(Ui.HORIZONTAL_LINE);
+    }
+
+    /**
+     * Prints all the tasks in the given task list.
+     *
+     * @param tasks The given task list.
+     */
+    public static void showTasks(List<Task> tasks) {
+        System.out.println(Ui.HORIZONTAL_LINE);
+        System.out.println(Ui.INDENTATION + "Here are the tasks in your list:");
+        if (tasks.isEmpty()) {
+            System.out.println(Ui.INDENTATION + "No tasks");
+        } else {
+            for (int i = 0; i < tasks.size(); ++i) {
+                System.out.println(Ui.INDENTATION + (i + 1) + ". " + tasks.get(i).toString());
+            }
+        }
+        System.out.println(Ui.HORIZONTAL_LINE);
+    }
+}
```

Comment:
I like how you've grouped all the relevant Ui methods appropriately under this class :)